.github/workflows/iroha2-dev-pr.yml

```diff
@@ -4,10 +4,10 @@ on:
   pull_request:
     branches: [iroha2-dev]
     paths:
-      - '**.rs'
-      - '**.json'
-      - '**.toml'
-      - '.github/workflows/**.yml'
+      - "**.rs"
+      - "**.json"
+      - "**.toml"
+      - ".github/workflows/**.yml"
```

Comment:
does this make a difference?

Reply:
I believe it would be safe to safely remove paths here so no it should not make a difference